Sources/sentry/include/SwiftFatalErrorMessage.h

```diff
@@ -1,32 +1,62 @@
-// SPDX-License-Identifier: BSD-3-Clause
-
-#ifndef sentry_native_include_Sentry_SwiftFatalErrorMessage_hh
-#define sentry_native_include_Sentry_SwiftFatalErrorMessage_hh
-
-#include <sentry.h>
-
-#ifdef _WIN32
-#include <atomic>
-
-inline void *getFatalErrorMessageHandle() {
-  // Docs: "The GetModuleHandle function returns a handle to a mapped module without incrementing its reference count"
-  // So we don't need to close it.
-  HMODULE hSwiftCore = GetModuleHandleA("swiftCore.dll");
-  if (!hSwiftCore) { return nullptr; }
-  auto pGetBuf = reinterpret_cast<std::atomic<const char *> *(*)()>(

```

Comment:
I also removed the cast here, as there were no atomic operations happening in this function.

Reply:
I think we still need this as this is a synchronization against other threads crashing with fatal errors.

Reply:
Sorry, not sure what you mean. This function just casts to an atomic before checking for null, but doesn't actually do any atomic loads or store, and then casts back to a void*, so I don't think it's necessary.